packages/services/src/state_listener.rs

```diff
@@ -64,7 +64,7 @@ where
                 // not included in block - check what happened to the tx
 
                 match (tx.state, self.l1_adapter.is_squeezed_out(tx.hash).await?) {
-                    (TransactionState::Pending, true) => {
+                    (TransactionState::Pending | TransactionState::IncludedInBlock, true) => {
```

Comment:
what could lead to the eth rpc reporting the response but not finding the tx by hash and are we sure that this represents a failed tx?